[ActionSheet] Fix behavior of `alwaysAlignTitleLeadingEdges`.

romoore opened the pull request:
The `alwaysAlignTitleLeadingEdges` API was incorrectly functioning as a
one-way switch that only worked after actions were added. Instead, it should
work at any point and be able to be disabled.

Fixes #8609

material-automation commented on components/ActionSheet/tests/snapshot/MDCActionSheetControllerSnapshotTests.m:
```diff
@@ -662,11 +662,33 @@ - (void)testActionSheetWhenActionsOnlyHaveTitlesAndAlignTitlesEnabled {
+                                                                  image:nil
+                                                                handler:nil];
+  MDCActionSheetController *controller =
+  [MDCActionSheetController actionSheetControllerWithTitle:nil];
```
clang-format suggested the following change:


```
-  [MDCActionSheetController actionSheetControllerWithTitle:nil];
+      [MDCActionSheetController actionSheetControllerWithTitle:nil];
```

<!-- identifier: clang-format -->
<!-- File: components/ActionSheet/tests/snapshot/MDCActionSheetControllerSnapshotTests.m -->

material-automation commented on components/ActionSheet/tests/snapshot/MDCActionSheetControllerSnapshotTests.m:
```diff
@@ -694,6 +716,29 @@ - (void)testActionSheetWhenSomeActionsHaveIconsAndTitlesAndSomeActionsOnlyHaveTi
 }
 
+- (void)
+testActionSheetWhenSomeActionsHaveIconsAndTitlesAndSomeActionsOnlyHavingTitlesAndAlignTitlesDisabled {
```
clang-format suggested the following change:


```
-testActionSheetWhenSomeActionsHaveIconsAndTitlesAndSomeActionsOnlyHavingTitlesAndAlignTitlesDisabled {
+    testActionSheetWhenSomeActionsHaveIconsAndTitlesAndSomeActionsOnlyHavingTitlesAndAlignTitlesDisabled {
```

<!-- identifier: clang-format -->
<!-- File: components/ActionSheet/tests/snapshot/MDCActionSheetControllerSnapshotTests.m -->

material-automation commented on components/ActionSheet/tests/snapshot/MDCActionSheetControllerSnapshotTests.m:
```diff
@@ -694,6 +716,29 @@ - (void)testActionSheetWhenSomeActionsHaveIconsAndTitlesAndSomeActionsOnlyHaveTi
+  MDCActionSheetAction *action1 =
+  [MDCActionSheetAction actionWithTitle:kShortTitle1Latin
+                                  image:[UIImage mdc_testImageOfSize:CGSizeMake(24, 24)]
+                                handler:nil];
```
clang-format suggested the following change:


```
-  [MDCActionSheetAction actionWithTitle:kShortTitle1Latin
-                                  image:[UIImage mdc_testImageOfSize:CGSizeMake(24, 24)]
-                                handler:nil];
+      [MDCActionSheetAction actionWithTitle:kShortTitle1Latin
+                                      image:[UIImage mdc_testImageOfSize:CGSizeMake(24, 24)]
+                                    handler:nil];
```

<!-- identifier: clang-format -->
<!-- File: components/ActionSheet/tests/snapshot/MDCActionSheetControllerSnapshotTests.m -->

material-automation commented on components/ActionSheet/tests/snapshot/MDCActionSheetControllerSnapshotTests.m:
```diff
@@ -694,6 +716,29 @@ - (void)testActionSheetWhenSomeActionsHaveIconsAndTitlesAndSomeActionsOnlyHaveTi
+                                                                  image:nil
+                                                                handler:nil];
+  MDCActionSheetController *controller =
+  [MDCActionSheetController actionSheetControllerWithTitle:nil];
```
clang-format suggested the following change:


```
-  [MDCActionSheetController actionSheetControllerWithTitle:nil];
+      [MDCActionSheetController actionSheetControllerWithTitle:nil];
```

<!-- identifier: clang-format -->
<!-- File: components/ActionSheet/tests/snapshot/MDCActionSheetControllerSnapshotTests.m -->

material-automation commented:
clang-format recommended changes.

Consider installing [git-clang-format](https://github.com/llvm-mirror/clang/blob/master/tools/clang-format/git-clang-format) and running the following command:

```bash
git clang-format $(git merge-base origin/develop HEAD)
```

<!-- identifier: clang-format -->

material-automation commented:
bazel detected changes to the following targets:

```
//components/ActionSheet:snapshot_tests
//components/ActionSheet:snapshot_tests_test_bundle
//components/ActionSheet:unit_tests
//components/ActionSheet:unit_tests_IPAD_PRO_12_9_IN_9_3
//components/ActionSheet:unit_tests_IPAD_PRO_12_9_IN_9_3_test_bundle
//components/ActionSheet:unit_tests_IPHONE_7_PLUS_IN_10_3
//components/ActionSheet:unit_tests_IPHONE_7_PLUS_IN_10_3_test_bundle
//components/ActionSheet:unit_tests_IPHONE_X_IN_11_0
//components/ActionSheet:unit_tests_IPHONE_X_IN_11_0_test_bundle
```

<!-- identifier: affected-targets -->

Commit:
[ActionSheet] Fix behavior of `alwaysAlignTitleLeadingEdges`.

The `alwaysAlignTitleLeadingEdges` API was incorrectly functioning as a
one-way switch that only worked after actions were added. Instead, it should
work at any point and be able to be disabled.

Fixes #8609

Commit:
Fix disabling the flag.

Commit:
Fix setting before adding actions

Commit:
Update test comment to be more explicit about why we order operations.

Commit:
Fix LFS

Commit:
Remove images.

Commit:
Re-add snapshot tests

Commit:
Revert "Fix LFS"

This reverts commit 6c2b66ead593e8713a0cac2b941ddb93b6b31a30.
